Enterprise-CMCS/macpro-mdct-mcr
# Remove MLR Get Started Page

### Description

There is no description for the MLR report in the `Get Started` page, so we're just removing it altogether.

### Related ticket(s)

N/A

---
### How to test

- Log in as a state user
- Click on `Enter MLR`

You should navigate directly to the MLR dashboard ✨ 

### Important updates

N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
